src/utils/narrow.js

```diff
@@ -384,6 +384,12 @@ export const emailsOfGroupPmNarrow = (narrow: Narrow): string[] =>
     },
   });
 
+/**
+ * The "PM key recipients" for a PM narrow; else error.
```

Comment:
Yeah -- the intention is to refer to that, but it's not the same type because it only has the emails.

This isn't very clear, so I'll add something to the jsdoc to clarify.